### PR TITLE
Rectification Of Typographical Error in design.md

### PR DESCRIPTION
**Pull request by iamrajiv**
Rectification Of Typographical Error in design.md

### CODE REVIEW

**Comment by iamrajiv** [comment]
@certik can you review this PR is this PR ready to be merged

File: doc/design.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ The function signature of `add` is:
 
 For more complicated expressions, instead of `add`, we might also consider
 using the naming scheme proposed in [1]. Another advantage of this approach is
-that compiler errors are much easier to understand, since it is either finding
+that compiler errors are much easier to understand since it is either finds
```
Comment (certik) [inline doc/design.md]: since it either finds

Comment (iamrajiv) [reply doc/design.md]: i have resolved the issue can you take a look @certik

Comment (iamrajiv) [reply doc/design.md]: I have resolved the issue can you take a look @certik

Comment (iamrajiv) [reply doc/design.md]: I have resolved the issue can you take a look @certik thanks for the suggestion :relaxed:

**Comment by iamrajiv** [comment]
@ShikharJ can review this PR

**Comment by iamrajiv** [comment]
@ShikharJ can you review this PR

**Comment by certik** [comment]
Thank you @iamrajiv.

**Comment by iamrajiv** [comment]
thanks for guidance @certik :blush:

**Pull request merged**